package.json

```diff
@@ -170,6 +170,7 @@
     "react-externs": "0.13.6",
     "react-with-direction": "1.3.1",
     "regexp.escape": "1.1.0",
+    "renovate": "20.2.0",
```

Comment:
Another blocker for this PR (in case my code review wasn't troublesome enough!)

Installing the `renovate` package requires `yarn` v1.17.0 or higher. Current stable is v1.22.4, but Travis default is v1.15.2.

```sh
error renovate@20.2.0: The engine "yarn" is incompatible with this module. Expected version "^1.17.0". Got "1.15.2"
```

We've asked Travis in the past about upgrading their built-in version of `yarn`, but they've refused. See https://github.com/travis-ci/travis-ci/issues/9445 and https://travis-ci.community/t/upgrade-yarn-to-a-newer-version/2135

Some alternatives:
- Figure out a way to run this check without installing the `renovate` package. Possible? We'll lose the benefit of renovate renovating renovate. (Heh!)
- Manually install `yarn` stable on Travis' Xenial VMs. We used to do this before adopting Xenial in #20987, but stopped because Xenial was new enough. I think it's okay to reinstate this once again.
https://github.com/ampproject/amphtml/blob/5e048d846ec4f196617da8c6c892b1dac42ecde2/.travis.yml#L15-L17
- Upgrade from Travis CI's Xenial to [Bionic](https://docs.travis-ci.com/user/reference/overview/#linux). Probably too much effort..

Reply:
I saw this failure in the last minutes of Thursday evening, and happily punted to this week :laughing: Thanks for offering some solutions! I think manually installing `yarn` looks to be the most robust choice here, provided it doesn't add much to build times (which I doubt)